tests/functional/spec/interstitial.spec.js

```diff
@@ -17,6 +17,7 @@ describe('Interstitial Pages', function () {
 
         // When
         expect(InterstitialPage.getMainHeading()).to.contain('Breakfast Interstitial Page')
```

Comment:
From http://martinfowler.com/bliki/GivenWhenThen.html

- The given part describes the state of the world before you begin the behavior you're specifying in this scenario. You can think of it as the pre-conditions to the test.
- The when section is that behavior that you're specifying.
- Finally the then section describes the changes you expect due to the specified behavior.

Maybe the BDD scenario is what is making this confusing, maybe it should be...

Given I am on the interstitial page
When I continue
Then I should go to the next question

Then it would be split out like...

```
//Given
startQuestionnaire(interstitial_schema)
InterstitialSurveyPage.setBreakfastFood('Cereal').submit()

// When
InterstitialPage.submit()

// Then
expect(NextPage.isOpen()).to.be.true
```